src/App.js

```diff
@@ -52,17 +58,34 @@ class App extends Component {
       .catch(error => console.log(error));
   }
 
+  openHomePage = () => {
+    this.setState({
+      pageStatus: 'home'
+    });
+  }
+
   render() {
-    return (
-      <div>
-        <h1>Mark My Parks</h1>
-        <ParkMap parks={this.state.parks}/>
-        <button onClick={this.showAllParks}>Show All Parks</button>
-        <button onClick={this.showVisitedParks}>Show Visited Parks</button>
-        <button onClick={this.showBucketList}>Show Bucket List Parks</button>
-        <FilterControls usStates={this.state.usStates}/>
-      </div>
-    )
+    switch(this.state.pageStatus) {
```

Comment:
Great use of the switch statement here for the landing page. Really like how it looks too!

Reply:
Good stuff! I definitely need to look into switch statements a little more, but it looks good!